src/rpc/blockchain.cpp

```diff
@@ -1359,6 +1360,7 @@ RPCHelpMan getdeploymentinfo()
             UniValue deploymentinfo(UniValue::VOBJ);
             deploymentinfo.pushKV("hash", blockindex->GetBlockHash().ToString());
             deploymentinfo.pushKV("height", blockindex->nHeight);
+            deploymentinfo.pushKV("script_flags", FormatScriptFlags(GetBlockScriptFlags(*blockindex, chainman)));
```

Comment:
This should not be a comma-deliminated string, but an array of strings.

Reply:
Huh, had thought about that previously and didn't like it. Reconsidering it, now I do. :shrug: